components/bounty/pull-and-proposal-hero/view.tsx

```diff
@@ -82,7 +84,8 @@ export default function PullAndProposalHeroView({
           <div className="col-auto">
             <PriceConversor
               currentValue={issueAmount}
-              currency={transactionalTokenSymbol || t("common:misc.token")}
+              currency={token.symbol || t("common:misc.token")} 
+              token={token}            
```

Comment:
You don't need the `currency` prop, the whole token is already being passed as `token` prop.